.github/workflows/android.yaml

```diff
@@ -104,5 +104,5 @@ jobs:
         uses: svenstaro/upload-release-action@04733e069f2d7f7f0b4aebc4fbdbce8613b03ccd # v2.9.0
         with:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
-          file: "Android APKs.zip"
+          file: "CoAT APKs.zip"
```

Comment:
I still think putting "non-Evergreen" in the name would make it clear that these APKs are not EG AOSP APKs, especially since they're placed under Evergreen releases on Github. Is it possible to rename them as `Non-Evergreen CoAT APKs.zip`? Or something similar, as long as it has `non-Evergreen` in the name. Thanks.